.github/workflows/node.js.yml

```
@@ -55,6 +55,26 @@ jobs:
             - run: npm run testCompile
             - run: npm run lint
 
+    jscpd:
+        needs: lint-commits
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                node-version: [18.x]
+        env:
+            NODE_OPTIONS: '--max-old-space-size=8192'
+        steps:
+            - name: check out repo
+              uses: actions/checkout@v4
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v4
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - name: install jscpd
+              run: npm install jscpd
```

it's often more readable not to have a name, when the "run" part gives the same (or better) information.
```suggestion
            - uses: actions/checkout@v4
            - name: Use Node.js ${{ matrix.node-version }}
              uses: actions/setup-node@v4
              with:
                  node-version: ${{ matrix.node-version }}
            - run: npm install jscpd
```